### PR TITLE
Add status badges

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
Add CI status badges to README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # GDASApp
 Global Data Assimilation System Application
+
+[![Testing on Orion](https://github.com/NOAA-EMC/GDASApp/actions/workflows/orion.yaml/badge.svg)](https://github.com/NOAA-EMC/GDASApp/actions/workflows/orion.yaml)
```
Comment (aerorahul) [inline README.md]:
```suggestion
[![Orion](https://github.com/NOAA-EMC/GDASApp/actions/workflows/orion.yaml/badge.svg)](https://github.com/NOAA-EMC/GDASApp/actions/workflows/orion.yaml)
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
+[![Run Unit Tests](https://github.com/NOAA-EMC/GDASApp/actions/workflows/unittests.yaml/badge.svg)](https://github.com/NOAA-EMC/GDASApp/actions/workflows/unittests.yaml)
```
Comment (aerorahul) [inline README.md]:
```suggestion
[![Unit Tests](https://github.com/NOAA-EMC/GDASApp/actions/workflows/unittests.yaml/badge.svg)](https://github.com/NOAA-EMC/GDASApp/actions/workflows/unittests.yaml)
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,6 @@
...
+[![Check Python Coding Norms](https://github.com/NOAA-EMC/GDASApp/actions/workflows/pynorms.yaml/badge.svg)](https://github.com/NOAA-EMC/GDASApp/actions/workflows/pynorms.yaml)
```
Comment (aerorahul) [inline README.md]:
```suggestion
[![Python Coding Norms](https://github.com/NOAA-EMC/GDASApp/actions/workflows/pynorms.yaml/badge.svg)](https://github.com/NOAA-EMC/GDASApp/actions/workflows/pynorms.yaml)
```

**Review by aerorahul — COMMENTED** [review]
Is it possible to do this on different lines:
e.g.
|    |   |
| --|  -- |
| Unit Tests | Python Coding Norms |
| Orion | Hera |

(Ignore the table)

**Comment by CoryMartin-NOAA** [comment]
@aerorahul your suggestion didn't work but I've changed the test names in this PR so they will update as appropriate when this is merged.

**Review by aerorahul — APPROVED** [review]

**Pull request merged**